specification/contosowidgetmanager/data-plane/Azure.Contoso.WidgetManager/preview/2022-11-01-preview/widgets.json

```diff
@@ -417,7 +414,29 @@
       ],
       "x-ms-enum": {
         "name": "OperationState",
-        "modelAsString": true
+        "modelAsString": true,
+        "values": [
```

Comment:
Why did this change?

Reply:
So this probably wouldn't be considered a "breaking change", because while it is a change to autorest output, it's not a "breaking" change to the autorest, since it's just documentation?

Reply:
Yes, adding documentation is not a breaking change